src/sentry/monitors/utils.py

```diff
@@ -18,3 +21,51 @@ def signal_first_monitor_created(project: Project, user, from_upsert: bool):
         first_cron_monitor_created.send_robust(
             project=project, user=user, from_upsert=from_upsert, sender=Project
         )
+
+
+def get_alert_rule(project: Project, user: User, monitor: Monitor, alert_rule: dict):
+    target_identifier = alert_rule.get("targetIdentifier", user.id)
+    target_type = alert_rule.get("targetType", "Member")
+    return {
+        "actionMatch": "any",
+        "actions": [
+            {
+                "id": "sentry.mail.actions.NotifyEmailAction",
+                "name": f"Send a notification to {target_type}",
+                "targetIdentifier": target_identifier,
+                "targetType": target_type,
+            }
```

Comment:
Since I think we wanted to support multiple notified users, should we go ahead and have the passed in thing be `targets` as an array? It's really not hard on the frontend to send a list of users / teams.